src/media/scripts/activities.jQuery.js

```diff
@@ -75,26 +71,36 @@
 			{
 				if (result.success != false)
 				{
-					replaceTemplate(result.data.results, theme, view, lang, result.data.total, ele);
+					replaceTemplate(result.data.results, result.data.total, ele);
 				}
 			}
 		});
 	}
 
-	replaceTemplate = function(activitiesData,theme,view, lang, total, ele){
-		var activityNumber = techjoomla.jQuery(ele).attr("activityNumber");
-		var start = techjoomla.jQuery(ele).attr("start");
+	replaceTemplate = function(activitiesData, total, ele){
```

Comment:
The variable replaceTemplate seems to be never declared. Assigning variables without defining them first makes them global. If this was intended, consider making it explicit like using window.replaceTemplate.